static/js/composebox_typeahead.js

```diff
@@ -998,7 +999,7 @@ function get_header_text() {
         default:
             return false;
     }
-    return "<em>" + tip_text + "</em>";
+    return `<em>${_.escape(tip_text)}</em>`;
```

Comment:
Are we double-escaping this?  (The values of tip_text come from i18n.t calls.)

Reply:
There’s nothing about these `i18n.t` calls that indicates them to be inherently HTML; indeed, the `_text` indicates otherwise. The rule needs be that we escape anything that’s not inherently HTML when inserting it into something that is.

Reply:
I was confused...the double-escaping with i18n happens with translation tags, which makes sense.